cli/azd/pkg/commands/pipeline/github_provider.go

```diff
@@ -431,10 +474,96 @@ func (p *GitHubCiProvider) configureConnection(
 		}
 	}
 
-	console.Message(ctx, fmt.Sprintf(
-		`GitHub Action secrets are now configured.
-		See your .github/workflows folder for details on which actions will be enabled.
-		You can view the GitHub Actions here: https://github.com/%s/actions`, repoSlug))
+	return nil
+}
+
+// Configures Github for OIDC authentication using registered application with federated identity credentials
+func (p *GitHubCiProvider) configureOidcAuth(ctx context.Context, azdEnvironment *environment.Environment, infraOptions provisioning.Options, repoSlug string, credentials json.RawMessage, console input.Console) error {
```

Comment:
Are we unnecessarily creating a secret on the SP in this case? Do we need a secret on the SP?

Reply:
The SP with password is created higher up in the pipeline config and we are just adding the federated identity.  We could "remove" the password, but lets chat as a group.